.circleci/config.yml

```diff
@@ -9,13 +9,13 @@ workflows:
             - build
           filters:
             branches:
-              ignore: /.*/
+              only: master
             tags:
-              only: /v[0-9]+(\.[0-9]+)*/
+              only: '/^v[\d.]+$/'
```

Comment:
I think this would match `1....0..1` 
(tho I doubt anyone would check in a version like that)

Reply:
Fixed the regex.